### PR TITLE
feat(pci.k8s.reset): add network proxy form

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-k8s-add_add-network-proxy-form_TAPC-251`
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #TAPC-87
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/network-form.template.html [hunk 1]
```diff
@@ -132,6 +132,7 @@
             ></span>
         </p>
         <oui-field
+            data-force-error-display="true"
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/network-form.template.html:135]: This was dropped, the error will now be displayed if the field isn't empty.

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 2]
```diff
@@ -77,93 +80,95 @@
...
+                            data-min="1"
                         />
-                    </oui-field>
+                    </div>
+                </div>
+                <div>
+                    <div>tcpfin timeout</div>
+                    <div>
+                        <oui-field
+                            data-force-error-display="true"
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:126]: Same as https://github.com/ovh/manager/pull/10940#discussion_r1492306465

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 3]
```diff
@@ -77,93 +80,95 @@
...
+                                data-max="$ctrl.maxTimeout"
+                            />
+                        </oui-field>
+                    </div>
+                </div>
+                <div>
+                    <div>tcp timeout</div>
+                    <div>
+                        <oui-field
+                            data-force-error-display="true"
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:143]: Same as https://github.com/ovh/manager/pull/10940#discussion_r1492306465

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 4]
```diff
@@ -77,93 +80,95 @@
...
+                                data-max="$ctrl.maxTimeout"
+                            />
+                        </oui-field>
+                    </div>
+                </div>
+                <div>
+                    <div>udp timeout</div>
+                    <div>
+                        <oui-field
+                            data-force-error-display="true"
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:160]: Same as https://github.com/ovh/manager/pull/10940#discussion_r1492306465

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 5]
```diff
@@ -51,10 +51,13 @@
...
-        <div class="container-fluid" data-ng-if="$ctrl.isIPTablesMode">
-            <div class="row">
-                <div class="col-xs-12 col-sm-2">min sync period</div>
-                <div class="col-xs-12 col-sm-2">
+        <div
+            class="kubernetes-proxy-form-field"
+            data-ng-if="$ctrl.isIPTablesMode"
+        >
+            <div>
+                <div>min sync period</div>
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:59]: Hardcoded no translation need ?

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 6]
```diff
@@ -63,9 +66,9 @@
                     />
                 </div>
             </div>
-            <div class="row">
-                <div class="col-xs-12 col-sm-2">sync period</div>
-                <div class="col-xs-12 col-sm-2">
+            <div>
+                <div>sync period</div>
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:70]: Hardcoded no translation need ?

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html [hunk 7]
```diff
@@ -77,93 +80,95 @@
...
-            </div>
-            <div class="row">
-                <div class="col-xs-12 col-sm-2">tcpfin timeout</div>
-                <div class="col-xs-12 col-sm-10">
-                    <oui-field
-                        data-error-messages="{ validTimeout: ('kubernetes_proxy_form_ipvs_timeout_invalid'|translate) }"
-                    >
+            <div class="kubernetes-proxy-form-field">
+                <div>
+                    <div>min sync period</div>
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:89]: Hardcoded no translation need ?

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html (hunk 6 above)
Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:70]: Yes

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html (hunk 5 above)
Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:59]: Yes

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html (hunk 7 above)
Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.template.html:89]: Yes

**Review by JacquesLarique — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by brofik — APPROVED** [review]

**Pull request merged**